### PR TITLE
Adds High-Frequency Blades

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl: Manly Dwarf
add: Added the High Frequency Blade to Traitors, costs 8 telecrystals. Don't try to disarm it.
soundadd: Increased likelihood of us getting sued over parody by 500%
/:cl:

## About The Pull Request
This adds the High Frequency Blade to the Traitor Uplink. On paper, it's just a worse esword, deals no damage, doesn't even fit in backpacks, the one thing it does fit into is extremely obvious, it has a bright light, pulling it out plays a sound, the sprite is large and very animated, it's a bit of a mess. The tradeoff is that it's much funnier, giving you essentially the old Captain's Sabre (it was originally a child of it, too) except buffed up to instantly dismember a random limb besides the chest or head. It can't kill anyone, but why would you need to when you can slice and dice with style?
Also don't try to disarm it. It'll end poorly.
Use a multitool on it to turn the sword red, brazilian, and have it play a very short snippet of music every time you pick it up or drop it. There's a ten second delay to prevent this from being abused to violate ears.

## Why It's Good For The Game
Not really a necessary feature but I dunno it's something, I thought it'd be funny and I needed to use the sprites, they were over a year old.
It's meant to mostly annoy your victims rather than kill them, for laughs rather than efficiency.

Don't even check the code, just merge this. You know you want to.

### CODE REVIEW

**Comment by cris5311** [comment]
+1

why the fuck not. now it is even easier to nugget the clown!

**Comment by ghost** [comment]
Considering it can't be put in backpacks and is very obvious I'd say there is no reason for powergamers to take this weapon - that's a good thing. +1

**Comment by Manly-Dwarf** [comment]
I added a light to maximize obviousness.
This compiles and doesn't conflict, I'm on an up-to-date hippiestation branch. So ignore Travis I guess.

**Comment by yobob591** [comment]
+1 RULES OF NATURE

**Comment by G-erma** [comment]
+1 yes blease this sounds amazing

**Review by AnthonyJNgo — APPROVED** [review]
I didn't see any conflicts

File: hippiestation/code/game/objects/items/melee/hfblade.dm [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
+	var/brazil = FALSE
+	var/brightness = 5
+
+/obj/item/melee/hfblade/Initialize()
+	. = ..()
+	AddComponent(/datum/component/butchering, 30, 95, 5)
+	set_light(brightness)
+	START_PROCESSING(SSobj, src)
+	. = ..()
```
Comment (yoyobatty) [inline hippiestation/code/game/objects/items/melee/hfblade.dm:35]: wow, and this was approved?

**Pull request merged**